src/index.ts

```diff
@@ -1,42 +1,50 @@
 import scrypt = require('scrypt')
 
 import * as constants from './utils/constants'
-import { generateSeed } from './utils/helpers'
+import { generateSeed, toNetworkByte, computeTemplate } from './utils/helpers'
 import { templates, templateChars } from './utils/templates'
 
-export const generateKey = (name: string, password: string, namespace?: string, version?: string): Buffer => {
-  if (!namespace) {
+export const generateKey = (name: string, password: string, version: number = constants.MP_ALGORITHM_VERSION,
+  namespace?: string): Buffer => {
```

Comment:
Yeah, I originally did that as to not break the 120-column barrier on my IDE. Multiline function should work well in this case.